frontend/src/components/Reaction.js

```diff
@@ -4,9 +4,9 @@ import { handleReaction, checkHasReacted } from '../utils'
 import { saveAllPosts, saveAllUsers } from '../store/actions/actions'
 import useUser from '../utils/customHooks/useUser'
 
-const Reaction = props => {
-  const { id } = props.post
-  const { type, whoReacted, Icon } = props.reaction
+const Reaction = ({ post, reaction }) => {
+  const { id } = post
+  const { type, whoReacted, Icon } = reaction
   const [hasReacted, setHasReacted] = useState(false)
```

Comment:
What's the reasoning behind destructuring the props? 
I am not saying it is bad. Just provide explanation.

Reply:
Less code and you do not have to type all the time "props.".I guess maybe it is not that good in just destructure props components but loops